### ISSUE TITLE
Basic search yields no results from non-boosted fields

### ISSUE THREAD

**Issue by pameyer** [IQSS/dataverse#5153]
in docker-aio

```
[ERROR] Failures: 
[ERROR]   SearchIT.testIdentifier:646 JSON path data.total_count doesn't match.
Expected: <1>
  Actual: 0

[INFO] 
[ERROR] Tests run: 76, Failures: 1, Errors: 0, Skipped: 1
```

**Comment by pameyer** [comment]
courtesy of @pdurbin , this is sounding like it's more of a configuration issues with docker-aio than a code or test issue.

**Comment by pameyer** [comment]
Initial troubleshooting suggests that this is an interaction of the DOI configuration in docker-aio and setupIT (more specifically, DOI configuration needs to happen in the middle of setupIT (after deployment, before `post`) because `post` is trying, and failing, to publish things.

Not fully clear on which change since 738b5fd1cbbffb28b4a4c56561a6902729d43bbd and edff192275df861739bc56f002adc9bf8cd77c51 triggered a change in behavior.

**Comment by matthew-a-dunlap** [comment]
@pameyer it looks like its actually due to my update of the solrconfig, for some reason the boosting update is stopping results from coming through. I'm investigating currently.

**Comment by matthew-a-dunlap** [comment]
The boost configuration was using the old "text" catchall. Solr boost wasn't complaining about it but also didn't know what to do with it, so results that weren't one of the core boosted values were just getting no results. We didn't notice it earlier because until #4938 boosting was not actually enabled.

**Comment by matthew-a-dunlap** [comment]
For this issue we will need to redeploy the solrconfig. We also need to update the config on phoenix.

To test this issue, searching for matches on attributes outside the ones listed in the boost section of the solrconfig ("qf") should return results. For example searching a persistentId.

**Comment by pameyer** [comment]
@matthew-a-dunlap thanks for taking a look.  I'm surprised that a solrconfig change would affect docker-aio (where there was a failure) and phoenix (where there wasn't) differently though.

**Comment by pdurbin** [comment]
I'm starting to get slightly freaked out about how search is working these days.

> This meant that searches on strings that didn't match one of the core boosted values were just getting no results.

Are we saying that if a field isn't boosted it can't be found by basic search? Let's use "Production Place" as an example of a field that isn't boosted. I enter "Boston" as the "Production Place":

<img width="1208" alt="screen shot 2018-10-10 at 1 30 19 pm" src="https://user-images.githubusercontent.com/21006/46754742-29e50800-cc91-11e8-92a7-6ece9986871d.png">

If I do a search for "Boston", shouldn't I be able to find it? I can't:

<img width="1208" alt="screen shot 2018-10-10 at 1 35 20 pm" src="https://user-images.githubusercontent.com/21006/46754796-58fb7980-cc91-11e8-8f8f-aa6556fe0951.png">

To find the dataset, I have to target the field, search for "productionPlace:Boston" like this:

<img width="1208" alt="screen shot 2018-10-10 at 1 36 45 pm" src="https://user-images.githubusercontent.com/21006/46754865-8cd69f00-cc91-11e8-8af4-27fe7b44860a.png">

Does pull request #5163 return us back to normal where a search for "Boston" will find the dataset even through "Production Place" is not a boosted field?

**Comment by matthew-a-dunlap** [comment]
@pdurbin It returns it back to normal. It's broken in that instance because we needed to change the name of the boost catch-all field. Without it things were falling through the cracks.

**Comment by pdurbin** [comment]
@matthew-a-dunlap awesome. Great fix, then. I changed the title of this issue to "searching on non-boosted fields yields no results" to clarify what the issue is. Off to QA.

**Comment by matthew-a-dunlap** [comment]
@pameyer The reason for the discrepancy between phoenix and docker is that phoenix does not have solr deployment automated in any way and is running off an old version of the solrconfig.xml

**Comment by pameyer** [comment]
@matthew-a-dunlap That makes sense - thanks.  Works for me in docker-aio on 5153-solr-search-fix-9e226002b.
